client/src/components/Reviews.jsx

```diff
@@ -59,7 +59,7 @@ class Reviews extends React.Component {
         for (var i = 0; i < filteredreviews.length; i++) {
           filteredreviews[i]['readMoreSelected'] = true;
         }
-        console.log('filteredreviews: ', filteredreviews);
+        //console.log('filteredreviews: ', filteredreviews);
 
       })
       .catch(err => console.log('getCustomerReviews Error: ', err))
```

Comment:
Something to consider is what happens on the user side if the code in this catch block is executed.

Reply:
Good point. Made a note of it. Added it to do do list